quickstart/examples/prefix-cache-aware/ms-kv-events/values.yaml

```diff
@@ -9,54 +9,28 @@ modelArtifacts:
 routing:
   modelName: Qwen/Qwen3-0.6B
   servicePort: 8000
-
-  proxy:
-    image: "ghcr.io/llm-d/llm-d-routing-sidecar:v0.2.0-RC1"
-    secure: false
-    connector: nixlv2
-
   parentRefs:
     - group: gateway.networking.k8s.io
       kind: Gateway
       name: infra-kv-events-inference-gateway
 
+  proxy:
```

Comment:
proxy can be removed right?

Reply:
@Gregory-Pereira was this required until releases are synced or this is carrying on?

Reply:
No one has taught me how to use TLS with new routing sidecar - so we will need the insecure proxy configurations